java/client/src/main/java/glide/api/commands/PubSubClusterCommands.java

```diff
@@ -8,21 +8,22 @@
  *
  * @see <a href="https://redis.io/docs/latest/commands/publish/">Pub/Sub Commands</a>
  */
-public interface PubSubClusterCommands {
+public interface PubSubClusterCommands extends PubSubBaseCommands {
 
     /**
-     * Publishes message on pubsub channel in sharded mode.
+     * Publishes message on pubsub channel.
      *
      * @since Redis 7.0 and above.
```

Comment:
Since applicable to sharded only

Reply:
I have changed the parameter comment to mention setting sharded to true only works in Redis 7.0+.